apps/web/pages/api/cancel.ts

```diff
@@ -1,14 +1,14 @@
-import type { NextApiRequest } from "next";
+import type { NextApiRequest, NextApiResponse } from "next";
 
 import handleCancelBooking from "@calcom/features/bookings/lib/handleCancelBooking";
 import { getSession } from "@calcom/lib/auth";
 import { defaultResponder, defaultHandler } from "@calcom/lib/server";
 
-async function handler(req: NextApiRequest & { userId?: number }) {
+async function handler(req: NextApiRequest & { userId?: number }, res: NextApiResponse) {
```

Comment:
No need for the response object here. We just return the values. defaultResponder takes care of the rest.